packages/template-retail-react-app/app/pages/account/order-detail.jsx

```diff
@@ -170,8 +170,8 @@ const AccountOrderDetail = () => {
                             spacing={[0, 3]}
                             divider={
                                 <Divider
-                                    visibility={{base: 'hidden', lg: 'visible'}}
-                                    orientation={{lg: 'vertical'}}
+                                    visibility={{base: 'visible'}}
```

Comment:
This visibility should stay until smaller screen where the line of Order Data and Order Number broke down til 2 lines. Currently we set it to be visible only on lg, this would make the medium screen to not see this Divider despite they are still on the same line. E.g
Small screen
![image](https://github.com/user-attachments/assets/73e0787b-b516-4aef-bc2a-226bf103c881)
Medium screen
![image](https://github.com/user-attachments/assets/4882d58b-791d-41bc-b8dd-c3ed8fbff4f0)